src/pkg/data/consumers/governor/exporter_test.go

```diff
@@ -0,0 +1,64 @@
+package consumers
+
+import (
+	api "github.com/vmware-tanzu/cloud-native-security-inspector/src/api/v1alpha1"
+	openapi "github.com/vmware-tanzu/cloud-native-security-inspector/src/pkg/data/consumers/governor/go-client"
+	v1 "k8s.io/api/core/v1"
+	"os"
+	"testing"
+)
+
+var (
+	mockClient   *openapi.APIClient
+	clusterID    = "testingId"
+	apiToken     = "apiToken"
+	namespace    = "testingNamespace"
+	name         = "name"
+	image        = "image"
+	imageID      = "imageId"
+	replicaCount = 2
+)
+
+const (
+	testHost   = "clusterapi.swagger.io:80"
+	testScheme = "http"
+)
+
+func TestMain(m *testing.M) {
+	cfg := openapi.NewConfiguration()
+	cfg.AddDefaultHeader("testheader", "testvalue")
+	cfg.Host = testHost
+	cfg.Scheme = testScheme
+	mockClient = openapi.NewAPIClient(cfg)
+	retCode := m.Run()
+	os.Exit(retCode)
+}
+
+func TestSendReportToGovernorSuccess(t *testing.T) {
```

Comment:
Write test cases for -ve scenarios for SendReportToGovernor like TimedOut and Invalid url.